library/pom.xml

```diff
@@ -45,9 +45,15 @@
 			<artifactId>urlrewritefilter</artifactId>
 			<version>4.0.3</version>
 		</dependency>
+		
 		<!-- At the moment we're unpacking the webjars, so we don't want them bundled into the WAR
 		     but we still need them as dependencies so the plugin unpacks them, that's why they
 		     have a scope of runtime -->
+		<dependency>
```

Comment:
Move the dependency above the comment, the comment belongs to the next dependency.